src/H5I.c

```diff
@@ -43,6 +43,8 @@
 #include "H5Tpkg.h"      /* Datatypes                                */
 #include "H5VLprivate.h" /* Virtual Object Layer                     */
 
+#include "uthash.h" /* Hash table functionality                 */
```

Comment:
It would be _really_ good to have a comment (here, or somewhere) that this is an external piece of code we're including.

Reply:
Will do